.travis.yml

```diff
@@ -4,94 +4,62 @@
 # For use with the UnderStrap WordPress theme
 # @link https://github.com/understrap/understrap
 
-# Declare project language and PHP versions to test against.
-# @link https://docs.travis-ci.com/user/languages/php/
-language: php
 
-# Declare versions of PHP to use. Use one decimal max.
-php:
-    - "7.3"
-    - "5.6"
+# Declare virtual environment and operating system.
+# @link https://docs.travis-ci.com/user/reference/overview/#virtualization-environments
+os: linux
+dist: xenial
 
 # Ditch sudo and use containers.
 # @link http://docs.travis-ci.com/user/migrating-from-legacy/#Why-migrate-to-container-based-infrastructure%3F
 # @link http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure
 sudo: false
 
-# Declare which versions of WordPress to test against.
-# Also declare whether or not to test in Multisite.
-env:
-    # Trunk (current version in development is 5.3)
-    # @link https://github.com/WordPress/WordPress
-    - WP_VERSION=master WP_MULTISITE=0
+# Declare project language and PHP versions to test against.
+# @link https://docs.travis-ci.com/user/languages/php/
+language: php
+php:
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - '7.2'
+  - '7.3'
+  - '7.4'
```

Comment:
You may want to add a build against PHP `nightly` (= PHP 8) which is allowed to fail to use as an early warning system for the upcoming changes.

This may need some tweaking as I don't think the PHPCS dependencies will install on PHP 8 yet, though Parallel-Lint should be ok.

Reply:
You're right! I first included `nightly` but removed it as the `composer install` was throwing an error. I'll think about a tweak.

Reply:
Might this be a solution?
I added `-ignore-platform-reqs` for build against PHP `nightly`, so `composer install` does no longer throw an error because of PHPCS. This allows Parallel-Lint to run.
```
php:
  - '5.6'
  - '7.0'
  - '7.1'
  - '7.2'
  - '7.3'
  - '7.4'
  - nightly

matrix:
  fast_finish: true
  include:
    - php: 7.4
      env: PHPCS=1
  allow_failures:
    - php: nightly
  
# Use this to prepare the build for testing.
before_script:
    # Speed up build time by disabling Xdebug.
    - phpenv config-rm xdebug.ini || echo 'No xdebug config.'
    - if [ "$TRAVIS_PHP_VERSION" == "nightly" ]; then composer install --ignore-platform-reqs; fi
    - if [ "$TRAVIS_PHP_VERSION" != "nightly" ]; then composer install; fi
    # Install JSCS: JavaScript Code Style checker
    # @link http://jscs.info/
    - if [[ "$PHPCS" == 1 ]]; then npm install -g jscs; fi
    # Install JSHint, a JavaScript Code Quality Tool
    # @link https://jshint.com/docs/
    - if [[ "$PHPCS" == 1 ]]; then npm install -g jshint; fi
    - if [[ "$PHPCS" == 1 ]]; then wget https://develop.svn.wordpress.org/trunk/.jshintrc; fi
```